src/components/Scorad/Scorad.js

```diff
@@ -0,0 +1,116 @@
+import { useState } from "react";
+
+import BodyParts from "../BodyParts/BodyParts";
+import Symptoms from "../Symptoms/Symptoms";
+import SleepAndItch from "../SleepAndItch/SleepAndItch";
+import Result from "../Result/Result";
+import FormNav from "../FormNav/FormNav";
+
+function Scorad() {
+  const [part, setPart] = useState(1);
+  const [partsSums, setPartsSums] = useState(Array(3).fill(0));
+  const [scoradResult, setScoradResult] = useState(0);
+  const [sectionAPoints, setSectionAPoints] = useState(Array(13).fill(0));
+  const [sectionBPoints, setSectionBPoints] = useState(Array(6).fill(0));
+  const [sectionCPoints, setSectionCPoints] = useState(Array(2).fill(0));
```

Comment:
Moja propozycja struktury state'u dla aplikacji:
```suggestion
    const scoradFormData = {
    1: {
      step: 1,
      component: <BodyPart/>,
      formSum: 0,
      activeStep: true
    },
    2: {
      step: 2,
      component: <Symptoms/>,
      formSum: 0,
      activeStep: false
    },
    3: {
      step: 3,
      component: <SleepAndItch/>,
      formSum: 0,
      activeStep: false,
    },
  }
  const bodyPartsData = [{ bodyPart: 'head', bodyProportion: 4.5, id: 1, frontSide: true, backSide: true, checked: false }]

  const [bodyParts, setBodyParts] = useState(bodyPartsData);
  const [scoradForm, setScoradForm] = useState(scoradFormData);

```

Reply:
Nie jestem pewna, czy po zmianie state'ów dla poszczególnych części formularza nadal potrzebuję tego całościowego state'u. Obliczanie wyniku zrobiłam w jednym kroku, bez obliczania i zapisywania w osobnym state'cie sum dla poszczególnych części